### PR TITLE
adds swat mask death sounds

### PR DESCRIPTION
**Pull request by nesisknesis**
from #8757 that werent added

#### Changelog

:cl:  
soundadd: adds death sounds to swat masks
/:cl:

### CODE REVIEW

**Comment by yacabo** [comment]
The swat mask is a child of the sec mask which means it's inheriting on_mob_death() from the sec mask. So this PR is a bit redundant

File: code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -192,4 +192,8 @@
 
 /obj/item/clothing/mask/gas/sechailer/on_mob_death()
 	. = ..()
-	playsound(loc, pick('sound/voice/cpdeath/die1.ogg', 'sound/voice/cpdeath/die2.ogg', 'sound/voice/cpdeath/die3.ogg', 'sound/voice/cpdeath/die4.ogg'), 50, 0) //lost biosignal for protection team unit 4, remaining units contain 
+	playsound(loc, pick('sound/voice/cpdeath/die1.ogg', 'sound/voice/cpdeath/die2.ogg', 'sound/voice/cpdeath/die3.ogg', 'sound/voice/cpdeath/die4.ogg'), 50, 0) //lost biosignal for protection team unit 4, remaining units contain
+
+/obj/item/clothing/mask/gas/sechailer/swat/on_mob_death()
+	. = ..()
+	playsound(loc, pick('sound/voice/cpdeath/cdie1.ogg', 'sound/voice/cpdeath/cdie2.ogg', 'sound/voice/cpdeath/cdie3.ogg'), 50, 0)
```
Comment (JamieD1) [inline code/modules/clothing/masks/hailer.dm:199]: Pretty sure this is going to play the sound twice

File: code/modules/clothing/masks/hailer.dm [hunk 2]
```diff
@@ -192,4 +192,8 @@
 
 /obj/item/clothing/mask/gas/sechailer/on_mob_death()
 	. = ..()
-	playsound(loc, pick('sound/voice/cpdeath/die1.ogg', 'sound/voice/cpdeath/die2.ogg', 'sound/voice/cpdeath/die3.ogg', 'sound/voice/cpdeath/die4.ogg'), 50, 0) //lost biosignal for protection team unit 4, remaining units contain 
+	playsound(loc, pick('sound/voice/cpdeath/die1.ogg', 'sound/voice/cpdeath/die2.ogg', 'sound/voice/cpdeath/die3.ogg', 'sound/voice/cpdeath/die4.ogg'), 50, 0) //lost biosignal for protection team unit 4, remaining units contain
+
+/obj/item/clothing/mask/gas/sechailer/swat/on_mob_death()
+	. = ..()
```
Comment (Hopekz) [inline code/modules/clothing/masks/hailer.dm:198]:
```suggestion
```
¯ \ _(ツ)_ / ¯

**Comment by JamieD1** [comment]
@Idinuum

**Pull request closed without merging**